optimize/client/public/static/redirect.html

```diff
@@ -1,29 +0,0 @@
-<!DOCTYPE html>
```

Comment:
why was this deleted?

Reply:
To me it looked more like a workaround. I hope that as part of migration to MVC I've addressed the root cause of the problem (attempt to authorize access to static resources), so hopefully we no longer need it. Yet if there is a requirement to authorize access to static resources then we may want to get it back 🤞